docs/iris/src/whatsnew/contributions_2.3.0/bugfix_2019-Aug-30_circular_quiver_plot.txt

```diff
@@ -0,0 +1 @@
+* :meth:`iris.plot.quiver` now handles circular coordinates.
```

Comment:
@stephenworsley Could you use `~iris.plot.quiver` instead? As Sphinx will collapse the method name nicely...

Reply:
@stephenworsley Also, this should be `:func:`, as `iris.plot.quiver` is a function not a method.